ofcourse-templates/about.html

```diff
@@ -1,11 +1,11 @@
-<%inherit file="master.mak"/>
-
-<div class="jumbotron">
-<h1>Hello, world!</h1>
-<p>This is a wonderful experiment using Twitter Boostrap, Flask, and Mako to make awesome.</p>
-<p><a href="#" class="btn btn-primary btn-lg">Learn more &raquo;</a></p>
-</div>
+{% extends "master.html" %}
 
+{% block jumbotron %}
+  <h1>Hello, world!</h1>
+  <p>This is a wonderful experiment using Twitter Boostrap, Flask, and Mako to make awesome.</p>
```

Comment:
There is the minor problem here that it still says "mako" here...in a jinja2 template. :100: